src/standalone/Soil/boundary_conditions.jl

```diff
@@ -626,12 +693,12 @@ function ClimaLand.∂tendencyBC∂Y(
         top = Operators.Extrapolate(),
     )
     K = Fields.level(interpc2f_op.(p.soil.K), face_len)
-    ϑ_l = Fields.level(interpc2f_op.(Y.soil.ϑ_l), face_len)
+    dψ = Fields.level(
+        interpc2f_op.(dψdϑ.(hydrology_cm, Y.soil.ϑ_l, ν, θ_r, S_s)),
```

Comment:
does it matter if we interpolate before or after the call to `dψdϑ`?

Reply:
it would in general matter if dψdϑ is nonlinear (and it is). I think we might still want to do it in this order (compute the field and then interpolate, instead of computing interpolated arguments and then computing the result), but im not sure. 

But, "interpc2f_op" just takes the top value as the face value, so for this particular use case (getting the top value of dψdϑ) it shouldnt matter, right?

Reply:
That makes sense to me